### PR TITLE
CLDR-18055 BRS 46 Update release note top row, add that row to downloads page

### PR DESCRIPTION
**Pull request by pedberg-icu**
CLDR-18055

- [x] This PR completes the ticket.

Update CLDR 46 Release Note top row info to what should be the final info for the release (links & date); add a row with the info to the overall Downloads/Releases page, and also update the row for Latest there.

We probably do not want to merge this until not long before things will be in their final locations for the release (spec, data files, release tag etc.). But would be good to have PR reviewed and test passings so PR is ready when needed. This is a site PR so on main branch.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/site/downloads/cldr-46.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: CLDR 46 Release Note
...
-|  46 | 2024-10-~~XX~~ |    ~~[v46]()~~   | ~~[CLDR46](http://unicode.org/Public/cldr/46/)~~ | [Charts46](http://unicode.org/cldr/charts/dev) |    [LDML46](http://www.unicode.org/reports/tr35/proposed.html)    | [Δ46](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246%22+ORDER+BY+priority+DESC) | ~~[release-46]()~~ |   [ΔDtd46](https://www.unicode.org/cldr/charts/dev/supplemental/dtd_deltas.html)  |   [46.0.0-BETA2](https://github.com/unicode-org/cldr-json/releases/tag/46.0.0-BETA2)  |
+|  46 | 2024-10-24 | [v46](/index/downloads/cldr-46) | [CLDR46](https://unicode.org/Public/cldr/46/) | [Charts46](https://unicode.org/cldr/charts/46/) | [LDML46](https://www.unicode.org/reports/tr35/tr35-73/tr35.html) | [Δ46](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2246%22+ORDER+BY+priority+DESC) | [release-46](https://github.com/unicode-org/cldr/releases/tag/release-46) | [ΔDtd46](https://www.unicode.org/cldr/charts/46/supplemental/dtd_deltas.html) | [46.0.0](https://github.com/unicode-org/cldr-json/releases/tag/46.0.0) |
```
Comment (stenshamn) [inline docs/site/downloads/cldr-46.md:9]: https://www.unicode.org/reports/tr35/tr35-73/tr35.html goes nowhere at this time. Expected?

**Comment by pedberg-icu** [comment]
Yes, several of the links will go nowhere until the final items are posted. Spec is one, data files another, final JSON data a third.

- Peter

File: docs/site/downloads/cldr-46.md (hunk 1 above)
Comment (srl295) [reply docs/site/downloads/cldr-46.md:9]: yes, expected, but working on it

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
Note: cloudflare failed because this was branched on the @pedberg-icu fork instead of unicode-org/cldr

**Comment by stenshamn** [comment]
I have a few nit-picks, but I can put those in a separate PR once this is merged.

**Pull request merged**